src/coreclr/tools/aot/ILCompiler.Compiler/IL/ILImporter.Scanner.cs

```diff
@@ -1259,6 +1259,53 @@ private void ImportBinaryOperation(ILOpcode opcode)
                     {
                         _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ThrowDivZero), "_divbyzero");
```

Comment:
```suggestion
                        _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.ThrowDivZero), "_divbyzero");
                        if (opcode == ILOpcode.rem)
                            _dependencies.Add(GetHelperEntrypoint(ReadyToRunHelper.Overflow), "_ovf");
```
Preexisting bug. Also, the same is missing for div above;